### PR TITLE
Options...

### PR DESCRIPTION
**Pull request by jchannon**
Pinging @davidfowl

### CODE REVIEW

**Comment by jchannon** [comment]
Pinging @alefranz

File: samples/CarterSample/Startup.cs [hunk 1]
```diff
@@ -17,7 +13,11 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddSingleton<IActorProvider, ActorProvider>();
 
-            services.AddCarter();
+            services.AddCarter(options => options.OpenApi = new OpenApiOptions("Carter <3 OpenApi", new[] { "http://localhost:5000" }, new Dictionary<string, OpenApiSecurity>
```
Comment (davidfowl) [inline samples/CarterSample/Startup.cs]: Don't make this an expression. It looks noisy and the line is too long.

Comment (jchannon) [reply samples/CarterSample/Startup.cs]: Pass an instance in instead?

Comment (davidfowl) [reply samples/CarterSample/Startup.cs]: I think so.

Comment (davidfowl) [reply samples/CarterSample/Startup.cs]: Wait, no, I meant the OpenAPI part.

Comment (jchannon) [reply samples/CarterSample/Startup.cs]: done on a new commit!

File: src/Carter/CarterOptions.cs [hunk 2]
```diff
@@ -88,8 +82,11 @@ public enum OpenApiIn
     public enum OpenApiSecurityType
     {
         apiKey,
```
Comment (davidfowl) [inline src/Carter/CarterOptions.cs]: Casing?

File: samples/CarterSample/Startup.cs [hunk 3]
```diff
@@ -26,37 +32,16 @@ public void Configure(IApplicationBuilder app, IConfiguration config)
             config.Bind(appconfig);
```
Comment (davidfowl) [inline samples/CarterSample/Startup.cs]: Configuration isn't being used here.

Comment (jchannon) [reply samples/CarterSample/Startup.cs]: Rejigged Configuration in the sample to use the new options approach in newer commit

**Pull request merged**